docs/fsharp/language-reference/import-declarations-the-open-keyword.md

```diff
@@ -27,7 +27,7 @@ open module-or-namespace-name
 ```
 
 ## Remarks
-Referencing code by using the fully qualified namespace or module path every time can create code that is hard to write, read, and maintain. Instead, you can use the `open` keyword for frequently used modules and namespaces so that when you reference a member of that module or namespace, you can use the short form of the name instead of the fully qualified name. This keyword is similar to the `using` keyword in C#, `using``namespace` in Visual C++, and `Imports` in Visual Basic.
+Referencing code by using the fully qualified namespace or module path every time can create code that is hard to write, read, and maintain. Instead, you can use the `open` keyword for frequently used modules and namespaces so that when you reference a member of that module or namespace, you can use the short form of the name instead of the fully qualified name. This keyword is similar to the `using` keyword in C#, `using`/`namespace` in Visual C++, and `Imports` in Visual Basic.
```

Comment:
Isn't it just a space between them when bringing everything into scope?

> using namespace \<namespace>

If so, I suggest ...

> \`using namespace\`